libp2p/protocols/pubsub/pubsub.nim

```diff
@@ -14,13 +14,22 @@ import pubsubpeer,
        ../protocol,
        ../../connection,
        ../../peerinfo
+import metrics
 
 export PubSubPeer
 export PubSubObserver
 
 logScope:
   topic = "PubSub"
 
+declareGauge(libp2p_pubsub_peers, "pubsub peer instances")
+declareGauge(libp2p_pubsub_topics, "current pubsub subscribed topics")
+declareGauge(libp2p_pubsub_validation_success, "current pubsub successfully validated messages")
```

Comment:
sorry for nitpicking, but do we really require `current pubsub` on every sentence, I'm guessing this labels are going to show up in whatever interface graphs them, can be cumbersome, but again, nitpicking here.

Reply:
Well it's a fair point, anyway everything is "current" so it's implied indeed: